extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/helper/SparkSQLLineageParseHelper.scala

```diff
@@ -94,6 +111,8 @@ trait LineageParser {
         k -> AttributeSet(attrs.collect {
           case attr if relationAttrSet.contains(attr) =>
             attr.withName(Seq(tableName, attr.name).filter(_.nonEmpty).mkString("."))
+          case attr if attr.name.contains(".") =>
+            attr
```

Comment:
what does this do ?

Reply:
For the lineage of the expression's subquery, the table name replacement are already generated during the expression parsing, so there is no need to replace the table name in the next